package/scripts/presto_client.py

```diff
@@ -80,7 +80,8 @@ def ensure_nodes_are_up(client):
         raise RuntimeError('Presto server failed to start within \
 {0} seconds.'.format(RETRY_TIMEOUT))
 
-
+# This class was copied more or less verbatim from
+# https://github.com/prestodb/presto-admin/blob/master/prestoadmin/prestoclient.py
```

Comment:
It would be nice to avoid copying code. Is it possible to upload prestoadmin to some python repositories and use presto-admin as dependencies?